internal/filesystem.go

```diff
@@ -485,3 +485,28 @@ func IsNetworkLinkUnmanaged(link string) bool {
 	}
 	return false
 }
+
+// PrefixCommonPath is supposed to be used for files which are version specific and not persistent
+func PrefixCommonPath(p string) string {
+	return prefixPath(p, "PREFIX_COMMON")
+}
+
+// PrefixDataPath is supposed to be used for files which are non version specific and persistent
```

Comment:
will there be any difference between this and the next function, because the description is the same?